tests/multi_ptr/multi_ptr_explicit_conversions.h

```diff
@@ -0,0 +1,162 @@
+              res_buf.template get_access<sycl::access_mode::write>(cgh);
+          auto val_acc =
+              val_buffer.template get_access<sycl::access_mode::read>(cgh);
+          cgh.single_task([=] {
+            const multi_ptr_t mptr_in(val_acc);
+            auto mptr_out =
+                sycl::multi_ptr<const T, source_space, decorated>(mptr_in);
+
+            // Check that second mptr has the same value as first mptr
+            res_acc[0] = *(mptr_out.get_raw()) == val_acc[0];
+          });
+        });
+      }
+      CHECK(res);
+    }
+  }
+};
+
+template <typename T>
+bool check_pointer_aliases(const std::string &type_name) {
+  SECTION(section_name("Check explicit pointer aliases")
+              .with("T", type_name)
+              .create()) {
+    {
+      INFO("raw_global_ptr");
+      STATIC_CHECK(std::is_same_v<
+                   sycl::raw_global_ptr<T>,
+                   sycl::multi_ptr<T, sycl::access::address_space::global_space,
+                                   sycl::access::decorated::no>>);
+    }
+    {
+      INFO("raw_local_ptr");
+      STATIC_CHECK(std::is_same_v<
+                   sycl::raw_local_ptr<T>,
+                   sycl::multi_ptr<T, sycl::access::address_space::local_space,
+                                   sycl::access::decorated::no>>);
+    }
+    {
+      INFO("raw_private_ptr");
+      STATIC_CHECK(
+          std::is_same_v<
+              sycl::raw_private_ptr<T>,
+              sycl::multi_ptr<T, sycl::access::address_space::private_space,
+                              sycl::access::decorated::no>>);
+    }
+    {
+      INFO("decorated_global_ptr");
+      STATIC_CHECK(std::is_same_v<
+                   sycl::decorated_global_ptr<T>,
+                   sycl::multi_ptr<T, sycl::access::address_space::global_space,
+                                   sycl::access::decorated::yes>>);
+    }
+    {
+      INFO("decorated_local_ptr");
+      STATIC_CHECK(std::is_same_v<
+                   sycl::decorated_local_ptr<T>,
+                   sycl::multi_ptr<T, sycl::access::address_space::local_space,
+                                   sycl::access::decorated::yes>>);
+    }
+    {
+      INFO("decorated_private_ptr");
+      STATIC_CHECK(
+          std::is_same_v<
+              sycl::decorated_private_ptr<T>,
+              sycl::multi_ptr<T, sycl::access::address_space::private_space,
+                              sycl::access::decorated::yes>>);
+    }
+  }
+}
+
+template <typename T>
+class check_multi_ptr_explicit_convert_for_type {
+ public:
+  void operator()(const std::string &type_name) {
+    check_pointer_aliases<T>(type_name);
+
+    const auto source_address_spaces = value_pack<
+        sycl::access::address_space, sycl::access::address_space::global_space,
+        sycl::access::address_space::local_space,
+        sycl::access::address_space::private_space>::generate_named();
+    ;
+    const auto is_decorated = multi_ptr_common::get_decorated();
```

Comment:
Would you mind to delete unnecessary statement?
```suggestion
    const auto source_address_spaces = value_pack<
        sycl::access::address_space, sycl::access::address_space::global_space,
        sycl::access::address_space::local_space,
        sycl::access::address_space::private_space>::generate_named();

    const auto is_decorated = multi_ptr_common::get_decorated();
```

Reply:
Removed.